binstar_client/inspect_package/ipynb.py

```diff
@@ -0,0 +1,33 @@
+import json
+import os
+
+
+class IPythonNotebook(object):
+    def __init__(self, filename, fileobj):
+        content = json.loads(fileobj.read())
+        self.name = content['metadata']['name'] or filename
```

Comment:
This needs to be 
`self.name = content['metadata']['name'] or self.basename`

Package names can not contain `/`